libs/transform.js

```diff
@@ -17,30 +18,33 @@ class TransformBase {
   /**
    * @param {string} species What this transform is called.
    * @param {string[]} requires What datasets are required before this can run?
+   * @param {string} produces What dataset does this transform produce?
    * @param {Boolean} input Does this transform require input?
-   * @param {Boolean} output Does this transform produce output?
+   * @param {Boolean} output Does this transform produce input?
```

Comment:
Is this PR up-to-date with master? I think we got rid of the `produces` field (we now rely on `report` blocks).